.gitignore

```diff
@@ -10,4 +10,5 @@ system-test/*key.json
 *.lock
 .DS_Store
 google-cloud-logging-winston-*.tgz
-google-cloud-logging-bunyan-*.tgz
\ No newline at end of file
+google-cloud-logging-bunyan-*.tgz
```

Comment:
aside: why are these two files listed here? What does logging-{winston,bunyan} have to do with compute.

Reply:
Because repo-tools does not have per-package overrides, and `.gitignore` is auto-generated. Somewhat similar issue is https://github.com/GoogleCloudPlatform/nodejs-repo-tools/issues/123.